Project.toml

```diff
@@ -12,7 +12,8 @@ HerbEvaluation = "eb1bf938-813d-4942-ac0f-b4657a683e76"
 HerbGrammar = "4ef9e186-2fe5-4b24-8de7-9f7291f24af7"
 Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
-StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
+StatsBase = "
 
```

Comment:
Logging appears two times in the Project.toml and the StatsBase dependency seems to have disappeared.